internal/librarian/generate.go

```diff
@@ -113,6 +114,10 @@ func newGenerateRunner(cfg *config.Config) (*generateRunner, error) {
 	if err != nil {
 		return nil, err
 	}
+	sourceRepo, err := gitrepo.NewRepository(&gitrepo.RepositoryOptions{Dir: cfg.APISource})
```

Comment:
Looks like the default behavior "If undefined, googleapis is the default and will be cloned to the output" is never implemented? Do you mind add check for empty cfg.APISource and implement default behavior here?

Or, I can also create a separate issue to track the fix.

Reply:
I actually filed that exact bug this morning: #1512